### DISCUSSION TITLE
Better console output for scan-only

### DISCUSSION THREAD

**Discussion by Lucent** [Breakthrough/DVR-Scan#157] [Ideas]
In trying combinations of --scan-only, --verbosity, --quiet, and --logfile, I wasn't able to create a great combination that'd serve my purpose of: find files with no motion to delete. Ideally, even with a wildcard, it could emit something like:

```
2024-03-24/1.mp4: 00:00:00.077,00:00:04.521
2024-03-24/2.mp4: NOMOTION
```

Now, it is difficult to get both the filename and a simple list of events or lack thereof. Only the ASCII art progress bar also shows filenames.

**Comment by Lucent** [comment]
This is my best attempt at a batch file that does what I want, but it double spaces some lines due to the options emitting nothing at all if there's no motion, but a trailing break if there is:

```
for /r "C:\videos" %%a in (*.mp4) do (
	echo(
	<nul set /p="%%a: "
	"dvr-scan.exe" -i "%%a" --bg-subtractor MOG2_CUDA --scan-only  --quiet --verbosity error
)
```

**Comment by Breakthrough** [comment]
If there are no motion events detected, you should see a log line similar to:

```
[DVR-Scan] No motion events detected in input.
```


You can [use `find` on the output of `dvr-scan`](https://www.dostips.com/forum/viewtopic.php?p=3843&sid=2746c8cf221c147980e6dedc2ebb0269#p3843) to determine if there was no motion or not:
```
dvr-scan.exe -i video.mp4 -so | find /i "No motion events detected in input."
if not errorlevel 1 (
   echo This line only runs if there was motion in the video.
)
